src/components/Demos.astro

```diff
@@ -0,0 +1,85 @@
+---
+import { Tag, Typography } from "@chainlink/blocks"
+import styles from "./Demos.module.css"
+
+export interface Demo {
+  image: string
+  title: string
+  description: string
+  tags: string[]
+}
+
+const demos: Demo[] = [
+  {
+    image: "/images/demos/Demos thumbnails 1.png",
+    title: "Custom Data Feed",
+    description:
+      "See how you can bring your financial data onchain in a variety of different ways by creating a Net Asset Value feed, Proof of Reserve feed, or other custom data feed.",
+    tags: ["CRE", "DATA FEEDS"],
+  },
+  {
+    image: "/images/demos/Demos thumbnails 2.png",
+    title: "Delivery vs. Payment (DvP)",
+    description:
+      "Learn how to sell a variety of assets via a Delivery vs. Payment transaction using the Chainlink Runtime Environment (CRE).",
+    tags: ["CRE"],
+  },
+  {
+    image: "/images/demos/Demos thumbnails 3.png",
+    title: "Digital Transfer Agent (DTA)",
+    description:
+      "Learn how to invest in a variety of funds via a Digital Transfer Agent (DTA) transaction using the Chainlink Runtime Environment (CRE).",
+    tags: ["CRE"],
+  },
+  {
+    image: "/images/demos/Demos thumbnails 4.png",
+    title: "Explore Chainlink",
+    description: "Explore how the Chainlink standard powers tokenization and onchain finance through",
+    tags: ["CRE", "CCIP", "DATA FEEDS"],
+  },
+]
+---
+
+<section class={styles.demosSection}>
+  <Typography variant="h2">Demos</Typography>
+  <div class={styles.grid}>
+    {
+      demos.map((demo) => (
+        <div class={styles.card}>
```

Comment:
The card are external links (as showm by the arrow shown on hover). Please change this to a Ancor and also add a link within the object